ui/src/components/textInputDialog.tsx

```diff
@@ -0,0 +1,71 @@
+import Button from "@mui/material/Button";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogTitle from "@mui/material/DialogTitle";
+import TextField from "@mui/material/TextField";
+import { ChangeEvent, ReactNode, useState } from "react";
+
+type textInputDialogProps = {
+  title: string;
+  titleId: string;
```

Comment:
I'm not sure if my previous comment got lost somewhere, but when you resolved it you didn't actually fix it. `titleId` is unused and should be removed.